mne/preprocessing/ica.py

```diff
@@ -1018,7 +1023,10 @@ def find_bads_eog(self, inst, ch_name=None, threshold=3.0,
         if len(scores) == 1:
             scores = scores[0]
 
-        return eog_idx, scores
+        if not hasattr(self, 'labels_'):
+            self.labels_ = {}
+        self.labels_['eog'] = list(eog_idx)
+        return self.labels_['eog'], scores
```

Comment:
corrmap defaults to 'bads'. I'm not sure about this, but maybe it'd make sense to unify all stuff that's primarily useful for a latter call to ica.apply( ... exclude= ...) under one default label.